implementations/microsoft/createVcJwt.js

```diff
@@ -0,0 +1,58 @@
+const {
+  CryptoBuilder,
+  KeyReference,
+  JoseBuilder,
+  CryptoFactoryNode,
+  KeyStoreInMemory,
+  Subtle,
+} = require("verifiablecredentials-crypto-sdk-typescript");
+const moment = require("moment");
+
+// See https://w3c.github.io/vc-data-model/#jwt-encoding
+const createVcJwt = async (credential, key) => {
```

Comment:
Consider using this, https://github.com/decentralized-identity/did-jwt-vc/blob/master/src/converters.ts#L80